dizoo/league_demo/league_scheduler.py

```diff
@@ -122,6 +123,12 @@ def update_para(self, cfg):
                     cfg.policy.learn.entropy_weight = min (self.factor*entropy_weight, self.change_range[1])
                 else:
                     cfg.policy.learn.entropy_weight = max (self.factor*entropy_weight, self.change_range[0])
+            
+            if self.schedule_mode == 'div':
+                if self.factor >= 1:
+                    cfg.policy.learn.entropy_weight = max (float(entropy_weight)/float(self.factor), self.change_range[0])
```

Comment:
(1) No space after function call `max`
(2) Need spaces before and after operator (`/` here)
(3) Same for the remaining places. Try using a style formatting tool to write code with good format.